README.md

```diff
@@ -70,3 +70,32 @@ import (
 ```
 
 See `cmd/hcp-sdk-go-client` for a complete example.
+
+## Contributing
+
+### Changelogs
+
+This repo requires that a chagnelog file be added in all pull requests. The name of the file must follow `[PR #].txt` and must reside in the `.changelog` directory. The contents must have the following formatting:
+
+~~~
+```release-note:TYPE
+ENTRY
+```
+~~~
+
+Where `TYPE` is the type of release note entry this is. This is one of either: `breaking-change`, `security`, `feature`, `improvement`, `deprecation`, `bug`.
+
```

Comment:
:sparkles: I love the thoroughness of explanation here on release-note types. The only potential knitpick I could suggest is maybe updating the default PR template to remind creating a changelog entry addition and listing as well the TYPE for releases. LGTM! :sunglasses: